packages/cli/src/templates/config.js

```diff
@@ -27,6 +27,10 @@ module.exports = {
 
   // Here you can configure the template that the variant preview gets embeded in.
   template: '${values.template}',
+  // <!-- uiengine:preview:start --> comments are used to display rendered components
+  // in code view
+  componentCommentDelimiter: true,
```

Comment:
I am not a fan of making this configurable. It adds unnecessary complexity to the code and makes it something that is hard to explain to the user. IMHO we should just add this feature and not have it optional.